Feature/create activity

MihaiDruta opened the pull request:
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Added functions for creating a new activity.

Commit:
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Commit:
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Added functions for adding an Activity.

Commit:
Deleted twig files.

SpartakusMd commented on src/DTO/ActivityDTO.php:
```diff
@@ -0,0 +1,108 @@
+     * @var string
+     * @Serializer\Type("string")
+     * @Assert\NotBlank(
+     *     message = "Activity name cannot be blank!")
```
If you start to use new lines in an annotation, make sure to put the closing parenthesis `)` on a new line.

SpartakusMd commented on src/DTO/TypeDTO.php:
```diff
@@ -0,0 +1,51 @@
+use DateTime;
+use JMS\Serializer\Annotation as Serializer;
+
+class TypeDTO
```
As we have disscussed, please rename the DTO and the entity. `Type` is too broad. Maybe `ActivityType` or something else.

SpartakusMd commented on src/DTO/TechnologyDTO.php:
```diff
@@ -0,0 +1,50 @@
+        $this->updatedAt = new DateTime();
+    }
+
+    public function getId(): ?int
```
Why you have this method if the field is public?

SpartakusMd commented on src/DTO/TypeDTO.php:
```diff
@@ -0,0 +1,51 @@
+        $this->updatedAt = new DateTime();
+    }
+
+    public function getId(): ?int
```
Why you have this method if the field is public?

Commit:
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Added custom exceptions EntityNotFound.

Commit:
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Fix migrations

Commit:
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Changed table names related to Type.
